config.json

```diff
@@ -1033,7 +1033,12 @@
       "core": false,
       "difficulty": 9,
       "slug": "pythagorean-triplet",
-      "topics": null,
+      "topics": [
+		"integer",
+		"lists",
+		"logic",
+		"mathematics"
+	  ],
```

Comment:
Sorry to be picky, but this square bracket should be indented one more space so that it is in line with `topics` and `unlocked_by`. And the the list of topics should be indented one more space so that they're two spaces in from `topics`. Does that make sense? It might help to look at entries for other exercises and copy their formatting :)